### PR TITLE
NGSTACK-951 add aria-label for social share links in footer

### PR DESCRIPTION
**Pull request by baljakl**

### CODE REVIEW

**Review by zulicek — APPROVED** [review]

File: translations/messages.de.yaml [hunk 1]
```diff
@@ -41,4 +41,11 @@ ngsite.job_application.deadline: 'Bewerbungsschluss'
 
 ngsite.skip_to_main_content: 'Zum Hauptinhalt springen'
 
+ngsite.visit_us_on_twitter: 'Besuchen Sie uns auf Twitter'
+ngsite.visit_us_on_linkedin: 'Besuchen Sie uns auf LinkedIn'
+ngsite.visit_us_on_facebook: 'Besuchen Sie uns auf Facebook'
+ngsite.visit_us_on_instagram: 'Besuchen Sie uns auf Instagram'
+
```
Comment (emodric) [inline translations/messages.de.yaml:48]: Please remove these extra white lines.

Comment (baljakl) [reply translations/messages.de.yaml:48]: removed extra white lines

Comment (emodric) [reply translations/messages.de.yaml:48]: Where? :smile:

Comment (baljakl) [reply translations/messages.de.yaml:48]: https://github.com/netgen/media-site/pull/223/commits/85f71eb6d8d9cdfdf70496a3847091e54f8442bb

**Review by emodric — APPROVED** [review]

**Pull request merged**